substrate-node/pallets/pallet-tft-price/src/tests.rs

```diff
@@ -227,9 +227,21 @@ fn test_parse_price() {
 fn test_parse_lowest_price_from_request() {
     let mut t = ExternalityBuilder::build();
     t.execute_with(|| {
-        let request_str = json_request_example();
+        let request_str = json_stellar_price_request_valid();
         let price = TFTPriceModule::parse_lowest_price_from_request(request_str).unwrap();
         assert_eq!(price, 33);
+
+        let request_str = json_stellar_price_request_empty();
```

Comment:
Looks great! I would not put it in the same test though... A unit test is supposed to test one specific use case. This test here is testing 3 different things. The problem when testing multiple things in one test is that when the test fails it is hard to know what exactly is failing unless you start analyzing the test. But as the test is bigger it is usually harder to analyze too. A good rule of thumb in my opinion is: one specific use case per unit test. This means you can usually split it up into:
1. Setup
2. Execution
3. Validation

Reply:
good practice indeed
thanks for sharing!